mouse-look.js

```diff
@@ -43,6 +45,14 @@ WL.registerComponent('mouse-look', {
             }
         }.bind(this));
 
+        if(this.pointerLockOnClick){
+            WL.canvas.addEventListener('mousedown', function() {
+                WL.canvas.requestPointerLock = WL.canvas.requestPointerLock ||WL.canvas.mozRequestPointerLock ||WL.canvas.webkitRequestPointerLock;
```

Comment:
```suggestion
                WL.canvas.requestPointerLock = WL.canvas.requestPointerLock || WL.canvas.mozRequestPointerLock || WL.canvas.webkitRequestPointerLock;
```

Unfortunately, we don't have a prettier config set up in this project yet, sorry about that 😅